include/lbann/layers/activations/tanh.hpp

```diff
@@ -64,7 +64,6 @@ class tanh_layer : public entrywise_activation_layer {
     // Activate GPU if needed
     if (cudnn != nullptr && T_layout == data_layout::DATA_PARALLEL) {
```

Comment:
I believe that the guard for only using the GPU if it is in data parallel mode should go away.